bofa_data.php

```diff
@@ -2185,6 +2185,1733 @@
+"<s>Killer</s> Festive Arc-Welding Elfbot" => [['Crimborg Collective Factory', '100', 5]],
+"<s>Killer</s> Festive Decal-Applying Elfbot" => [['Crimborg Collective Factory', '100', 5]],
+"<s>Killer</s> Festive Laser-Calibrating Elfbot" => [['Crimborg Collective Factory', '100', 5]],
+"<s>Killer</s> Festive Weapons-Assembly Elfbot" => [['Crimborg Collective Factory', '100', 5]],
+"Rudolphus of Crimborg" => [['Crimborg Collective Factory', '100', 5]],
+"alielf" => [['Domed City of Grimacia', '100', 6]],
+"cat-alien" => [['Domed City of Grimacia', '100', 6]],
+"dog-alien" => [['Domed City of Grimacia', '100', 6]],
+"grizzled survivor" => [['Domed City of Grimacia', '100', 6]],
+"unhinged survivor" => [['Domed City of Grimacia', '100', 6]],
+"whiny survivor" => [['Domed City of Grimacia', '100', 6]],
+"dogcat" => [['Domed City of Ronaldus', '100', 6]],
+"ferrelf" => [['Domed City of Ronaldus', '100', 6]],
+"hamsterpus" => [['Domed City of Ronaldus', '100', 6]],
+"overarmed survivor" => [['Domed City of Ronaldus', '100', 6]],
+"primitive survivor" => [['Domed City of Ronaldus', '100', 6]],
+"unlikely survivor" => [['Domed City of Ronaldus', '100', 6]],
+"judgmental eye" => [["Doubt Man's Level", '100', 3]],
+"reproachful heart" => [["Doubt Man's Level", '100', 3]],
+"work hat" => [["Doubt Man's Level", '100', 3]],
+"Doubt Man" => [["Doubt Man's Level", '100', 3, 'special']],
+"cold skeleton" => [['Dreadsylvanian Castle', '-1', 10]],
+"cold vampire" => [['Dreadsylvanian Castle', '-1', 10]],
+"hot skeleton" => [['Dreadsylvanian Castle', '-1', 10]],
+"hot vampire" => [['Dreadsylvanian Castle', '-1', 10]],
+"sleaze skeleton" => [['Dreadsylvanian Castle', '-1', 10]],
+"sleaze vampire" => [['Dreadsylvanian Castle', '-1', 10]],
+"spooky skeleton" => [['Dreadsylvanian Castle', '-1', 10]],
+"spooky vampire (Dreadsylvanian)" => [['Dreadsylvanian Castle', '-1', 10]],
+"stench skeleton" => [['Dreadsylvanian Castle', '-1', 10]],
+"stench vampire" => [['Dreadsylvanian Castle', '-1', 10]],
+"The Unkillable Skeleton" => [['Dreadsylvanian Castle', '-1', 10, 'special']],
+"Count Drunkula" => [['Dreadsylvanian Castle', '-1', 10, 'special']],
+"cold ghost" => [['Dreadsylvanian Village', '-1', 10]],
+"cold zombie" => [['Dreadsylvanian Village', '-1', 10]],
+"hot ghost" => [['Dreadsylvanian Village', '-1', 10]],
+"hot zombie" => [['Dreadsylvanian Village', '-1', 10]],
+"sleaze ghost" => [['Dreadsylvanian Village', '-1', 10]],
+"sleaze zombie" => [['Dreadsylvanian Village', '-1', 10]],
+"spooky ghost (Dreadsylvanian)" => [['Dreadsylvanian Village', '-1', 10]],
+"spooky zombie" => [['Dreadsylvanian Village', '-1', 10]],
+"stench ghost" => [['Dreadsylvanian Village', '-1', 10]],
+"stench zombie" => [['Dreadsylvanian Village', '-1', 10]],
+"Mayor Ghost" => [['Dreadsylvanian Village', '-1', 10, 'special']],
+"Zombie Homeowners' Association" => [['Dreadsylvanian Village', '-1', 10, 'special']],
+"cold bugbear" => [['Dreadsylvanian Woods', '-1', 10]],
```

Comment:
i assume we are not using the data in such a way that the weird "-1" on dread stuff actually kicks our ass? i guess i will see this below lol

Reply:
actually i took a quick look and it does look like maybe this isn't handled? i believe this will manifest as -1% combat rate in line 4327 of bofa_data.php unless i'm missing something. i'd just add an if catch like you have on the other elements.

Reply:
Yes, it will, I decided to leave it as is for now.